.dockerignore

```
@@ -0,0 +1,5 @@
+.env
+.venv
+.git
+.vscode
+Dockerfile
```

A couple of additional suggestions:
```suggestion
Dockerfile
.pytest_cache
**/__pycache__
```
In case you're not aware, worth noting that `.dockerignore` has slightly different behaviour to `.gitignore`, in particular in that it _doesn't_ ignore subfolders by default - we need to explicitly reference them with the `**` prefix to mean arbitrary levels of nesting should be considered for this rule. I think that your existing entries wouldn't matters here, but this is one of those issues that can easily lead to weird confusion when some behaviour starts caching oddly!